src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1343,4 +1343,10 @@
   <data name="AssigningSymbolAndItsMemberInSameStatementTitle" xml:space="preserve">
     <value>Assigning symbol and its member in the same statement.</value>
   </data>
+  <data name="AvoidInfiniteRecursionMessage" xml:space="preserve">
+    <value>This call is leading to an infinite recursion.</value>
```

Comment:
Shall I keep a "generic" message in prevision for the next implementation or go for a specific message for the time being?